pkg/shipshape/shipshape.go

```diff
@@ -19,7 +19,7 @@ import (
 
 var RunConfig config.Config
 var RunResultList result.ResultList
-var OutputFormats = []string{"json", "junit", "simple", "table", "lagoon-facts"}
+var OutputFormats = []string{"json", "junit", "simple", "table", "lagoon-problems"}
```

Comment:
I believe this is not required anymore.
```suggestion
var OutputFormats = []string{"json", "junit", "simple", "table"}
```